ci/script.sh

```diff
@@ -455,6 +458,17 @@ main() {
             popd
 
             cargo check --manifest-path $td/Cargo.toml
+
+            # Test RISC-V K210
+            pushd $td
+
+            $cwd/target/$TARGET/release/svd2rust --target riscv -i $td/k210.svd
```

Comment:
```suggestion
            RUST_BACKTRACE=1 $cwd/target/$TARGET/release/svd2rust --target riscv -i $td/k210.svd
```

I've just issued #298 to get some feedback in case `svd2rust` crashes during CI. Mind adding that here as well?

Reply:
Sure!